cmd/lightclient/sentinel/sentinel.go

```diff
@@ -114,9 +119,36 @@ func (s *Sentinel) createListener() (*discover.UDPv5, error) {
 		return nil, err
 	}
 
+	go func() {
```

Comment:
We can merge this but I do not like this archittecture, I may change it in the future.